### PR TITLE
adds first cut of essentials list

### PR DESCRIPTION
**Pull request by svs**
Please note this is not about the front end. For the front end we will need to make commonly used items easy to find and enter. For example, potatoes and tomatoes are commonly used but they're under Vegetable->Staples.

In general I'm not happy with this taxonomy. Will need someone more familiar with this stuff to help out and make it better. Anyways, here's a first cut.

Will need to add Medicines and Toileterries/Sanitarywear also

### CODE REVIEW

File: docs/essentials_list.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - Cheese
+  - Breakfast Foods
+    - Eggs
+    - Poha/Suji
+  - Beverages
+    - Tea
+    - Coffee
+    - Soft Drinks
+  - Vegetables
+    - Staples
```
Comment (abhinavdalal) [inline docs/essentials_list.md]: should this really be multi level.. i have kinda a flat list of essentials..

File: docs/essentials_list.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    - Dahi
+    - Butter
+    - Cheese
+  - Breakfast Foods
+    - Eggs
+    - Poha/Suji
+  - Beverages
+    - Tea
+    - Coffee
+    - Soft Drinks
```
Comment (abhinavdalal) [inline docs/essentials_list.md]: Soft Drinks should surely not be essential !! but okay..

File: docs/essentials_list.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+- Food
+  - Flour
+    - Atta
+    - Besan
+    - Maida
+  - Bread
+  - Rice
+  - Pulses
+    - Daal
+    - Rajma / Chhole
```
Comment (abhinavdalal) [inline docs/essentials_list.md]: i think we need to split up rajma and chhole as 2 separate ones

File: docs/essentials_list.md (hunk 1 above)
Comment (svs) [reply docs/essentials_list.md]: Why? If Vegetables are not available why go vegetable by vegetable? If user wants to be precise they should be allowed to.

What advantages does a flat list have? Pls remember we can flatten the list for the frontend. But on the data model we want to have categories.

File: docs/essentials_list.md (hunk 3 above)
Comment (svs) [reply docs/essentials_list.md]: They serve the same purpose.

**Comment by svs** [comment]
We should add a "Kids" and a "Treats" section.

File: docs/essentials_list.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  - Staples
+    - Salt
+    - Sugar
+    - Oil
+    - Spices
+  - Dairy
+    - Milk
```
Comment (shivamsupr) [inline docs/essentials_list.md]: @svs we should add soy-milk as well.

File: docs/essentials_list.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
+- Food
+  - Flour
+    - Atta
+    - Besan
+    - Maida
+  - Bread
+  - Rice
```
Comment (shivamsupr) [inline docs/essentials_list.md]: Rice can be categorized too, 

- white rice
- brown rice

File: docs/essentials_list.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  - Pulses
+    - Daal
+    - Rajma / Chhole
+  - Staples
+    - Salt
+    - Sugar
+    - Oil
```
Comment (shivamsupr) [inline docs/essentials_list.md]: Oil can be categorized too,

- Refined oil
- Olive oil
- Coconut oil
- Ghee

File: docs/essentials_list.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    - Spices
+  - Dairy
+    - Milk
+    - Dahi
+    - Butter
+    - Cheese
+  - Breakfast Foods
+    - Eggs
+    - Poha/Suji
```
Comment (shivamsupr) [inline docs/essentials_list.md]: Makhana, dates, seeds should be added to this list.

**Pull request merged**